### ISSUE TITLE
It's possible return 0 when store xpath count find no elements

### ISSUE THREAD

**Issue by Yrds** [SeleniumHQ/selenium-ide#462]
Is there a way to return 0 when no elements are found?

For example:

store xpath count | xpath=td//img | imagesInTable

I want to know how many images are in table to assert after i change the state of system. When i have 1 or more, it runs flawlessly, but when there is no images, i am getting this error:

```
    TimeoutError: Waiting for at least one element to be located By(xpath, //td/img)
    Wait timed out after 15017ms

      38 |     element.click();
      39 |   });
    > 40 |   await driver.wait(until.elementsLocated(By.xpath(`//td/img`)), configuration.timeout);
         |                ^
      41 |   await driver.findElements(By.xpath(`//td/img`)).then(elements => {
      42 |     vars["fotosAntes"] = elements.length;
      43 |   });
```

All i want is to assert how many images i have before and after uploaded some photos. If you have tips to solve this problem without my solution, i will appreciate .

Edit: this happens only in selenium-side-runner, will open a bug report and give more details

**Comment by barancev** [comment]
@Yrds Why did you decide to close the report? This seems to be a legit issue.

**Comment by barancev** [comment]
Ah, it's a duplicate of #388

**Comment by Yrds** [comment]
@barancev Sorry, i forgot to mention it was fixed in 3.5.0 alpha:
https://github.com/SeleniumHQ/selenium-ide/commit/09d1d722a29bc44979fc26bbf86df39967d61f41
